docarray/typing/abstract_type.py

```diff
@@ -1,12 +1,16 @@
 from abc import abstractmethod
-from typing import Any, Type, TypeVar
+from typing import TYPE_CHECKING, Any, Type, TypeVar
 
 from pydantic import BaseConfig
 
 from docarray.utils._internal.pydantic import is_pydantic_v2
 
-if not is_pydantic_v2():
-    from pydantic.fields import ModelField
+if TYPE_CHECKING:
+    if not is_pydantic_v2():
```

Comment:
why dont u do as jina did for docarray v2, have a variable instead of a method call each time?

Reply:
yeah okay fair